### ISSUE TITLE
Dev Branch / Crash line 229 cuda_extra.cu / Windows 10 / 1070 GPU

### ISSUE THREAD

**Issue by tlorens** [fireice-uk/xmr-stak-nvidia#154]
Don't have the exact error in front of me.  But getting GPU 0: Bad-- something over time.   Debugger pops up and says it's on line 229 of cuda_extra.cu.    I'm hardly a C++ developer and even worse at this crypto+nVidia stuff.  

`CUDA_CHECK(ctx->device_id, cudaMemcpy( rescount, ctx->d_result_count, sizeof (uint32_t ), cudaMemcpyDeviceToHost ));`

Comparing this with what's in the master branch, it looks like a completely different approach to perform the operation.   I'd try to step through it, but like I said, I'm not much of a C++ guy, though I do understand basic C. (I'm a web-dev).

I'm willing to do take whatever steps necessary to provide you with more information.

**Comment by PrismIO** [comment]
Yeah this issue seems to be very common and no solution from what I can see,

**Comment by PrismIO** [comment]
My issue #175

**Comment by psychocrypt** [comment]
Should hopfully also solved with the next release. coming soon.

**Comment by tlorens** [comment]
I got around it by tweaking the GPU settings.   I don't recall exactly what the before/after values were.
